src/lib/pipes/beautify.pipe.ts

```diff
@@ -0,0 +1,13 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+    name: 'beautify'
```

Comment:
should have qGrid prefix, and can we do like that

`text | qGridText('fromCamelCase')` or something like that